frontend/src/modules/entitieslist/components/EntitiesList.tsx

```diff
@@ -1,5 +1,5 @@
 import React, { useRef, useEffect } from 'react';
-import InfiniteScroll from 'react-infinite-scroller';
```

Comment:
Ok It means, changes will have to occur with the yarn.lock file, hope you're play with that.

Reply:
That's expected. The diff in yarn.lock is huge, because the registry changed from npmjs.org to yarnpkg.com. I assume you'll need to change your yarn config to mitigate that.